### PR TITLE
:sparkles: Homepage revamp

### PR DESCRIPTION
**Pull request by mauromorales**
supersedes #65

### CODE REVIEW

**Review by mauromorales — COMMENTED** [review]
The hovered color for the community section gets a bit lost, maybe use black instead? or another different color 

image

File: static/js/trees.js [hunk 1]
```diff
@@ -0,0 +1,575 @@
+const DOWNLOAD_CARDS_TREE = [
+    {
+        integration:"ubuntu",
+        links: {
+            "X86": "https://github.com/kairos-io/provider-kairos/releases/download/v2.3.1/kairos-ubuntu-v2.3.1-k3sv1.25.11+k3s1.iso",
```
Comment (mauromorales) [inline static/js/trees.js:5]: all these need to be generated with our latest release variables in Hugo

**Comment by Itxaka** [comment]
cant see kairos name or the logo anywhere in the index

image


IMO mascot is good, but the logo should be on its place instead as its plastered everywhere and it shows it more. Also the Kairos name should be around, maybe on the navbar?

File: layouts/index.html [hunk 2]
```diff
@@ -33,42 +33,44 @@
...
               </h1>
-              <span class="subtitle astro-7XAARZHW">The immutable Linux meta-distribution <br class="astro-7XAARZHW"> for edge Kubernetes</span>
+              <span class="subtitle astro-7XAARZHW">
+                Transform any Linux system into a secure, <br />
+                customizable, and easily managed platform <br />
+                for edge computing.</span>
               <div class="buttons astro-7XAARZHW">
                 <a id="quickstart" href="/docs" class="astro-7XAARZHW">
-                  Documentation
+                  Quick Start
```
Comment (mauromorales) [inline layouts/index.html:47]: this one should point now to the quick start page instead of the docs

File: layouts/index.html [hunk 3]
```diff
@@ -33,42 +33,44 @@
...
                 </a>
-                <a id="download" href="https://github.com/kairos-io/provider-kairos/releases" class="astro-7XAARZHW">Download</a>
+                <a id="download" href="https://github.com/kairos-io/provider-kairos/releases" class="astro-7XAARZHW"><span class="github-star"><img src="/index/star.svg" alt="github star"></span>Star us on GitHub</a>
```
Comment (mauromorales) [inline layouts/index.html:49]: Should this point to the repo homepage instead?

File: layouts/index.html [hunk 4]
```diff
@@ -33,42 +33,44 @@
...
+            </div>
+
+            <div class="hero-logo astro-7XAARZHW">
+              <img src="/index/armadillo.png" alt="main logo" width="318" class="astro-7XAARZHW">
+              <div class="hero-latest-release">
+                Check out
+                <p>our latest release</p>
+                <div class="hero-latest-release-version">
+                  <div>Latest release</div>
+                  <div>{{$.Page.Site.Params.softwareVersions.kairos_version}}</div>
```
Comment (mauromorales) [inline layouts/index.html:60]: this is good, but seems like we forgot to update the version variable to 2.3.2

File: static/js/trees.js [hunk 5]
```diff
@@ -0,0 +1,575 @@
...
+                            div: {
+                                content: "Downloads"
+                            }
+                        }, ...DOWNLOAD_CARDS_TREE
+                    ]
+            }
+            },
+            {
+                a: {
+                    href: "https://github.com/kairos-io/kairos",
```
Comment (mauromorales) [inline static/js/trees.js:148]: let's point this to the image matrix https://kairos.io/docs/reference/image_matrix/#image-flavors

File: static/js/trees.js [hunk 6]
```diff
@@ -0,0 +1,575 @@
...
+                                src: imgSrc,
+                                alt: imgSrc
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                a: {
+                    href: link,
```
Comment (mauromorales) [inline static/js/trees.js:262]: can we configure so some of these links open in a new tab? @mirceabozi

File: static/js/trees.js [hunk 7]
```diff
@@ -0,0 +1,575 @@
...
+    },
+    {
+        content: "Check us out in this The New Stack article by our maintainer, Ettore di Giacinto",
+        imgSrc: "/index/press-blade/theNewStack.png",
+        link: "https://thenewstack.io/livin-kubernetes-on-the-immutable-edge-with-kairos-project/",
+    },
+    {
+        content: "Learn how to use the famous OSS project, LocalAI, with Kairos and K3s on your nodes",
+        imgSrc: "/index/press-blade/k8s-superpowers.png",
+        link: "https://kairos.io/docs/examples/localai/",
```
Comment (mauromorales) [inline static/js/trees.js:237]: @mudler is this the right link for k8s?

File: static/js/trees.js [hunk 8]
```diff
@@ -0,0 +1,575 @@
...
+                title: "Architecture",
+                imgSrc: "/index/basics/shell.png",
+                text: "Get inside Kairos, from the factory <br/> to P2P mesh capabilities.",
+                href: "https://kairos.io/docs/architecture/"
+            },
+            {
+                title: "Examples",
+                imgSrc: "/index/basics/origami-bird.png",
+                text: "Stretch your wings with best practices of common tasks after installing Kairos.",
+                href: "https://github.com/kairos-io/kairos/tree/master/examples"
```
Comment (mauromorales) [inline static/js/trees.js:468]: @mudler should this one point to the repo or to https://kairos.io/docs/examples/ ?

**Comment by mauromorales** [comment]
> cant see kairos name or the logo anywhere in the index. I mena, the first mention of Kairos is way below and cut.
> 
> image
> 
> IMO mascot is good, but the logo should be on its place instead as its plastered everywhere and it shows it more. Also the Kairos name should be around, maybe on the navbar?

Good point, there should be a logo on the nav bar 

cc @mirceabozi

**Comment by marcelaquintela** [comment]
Screenshot 2023-08-09 at 12 36 37

I'd suggest we make these 2 bottom boxes shorter to get rid of some empty space

**Comment by marcelaquintela** [comment]
Screenshot 2023-08-10 at 16 44 29
Minor thought but I'd make the button smaller (shorter on width) so the icon is closer to the CTA sentence

**Pull request closed without merging**